### ISSUE TITLE
Add a feature to apply user's smb.conf in secondary DC 

### ISSUE THREAD

**Issue by TsutomuNakamura** [hobby-labs/samba-ad-container#38]
Like #32.

- [ ] Add a feature to replace smb.conf by smb.conf.master after Samba has provisioned

**Comment by TsutomuNakamura** [comment]
For testing.

- Regression
```
# docker run --name pdc01 --hostname pdc01 -ti hobbylabs/samba-ad-container:develop
```

- New feature
```
# mkdir -p ./etc/samba
# docker run --name pdc01 --hostname pdc01 \
    --volume ${PWD}/etc/samba:/etc/samba -ti hobbylabs/samba-ad-container:develop
```

**Comment by TsutomuNakamura** [comment]
This feature already implemented at #33. I should test it.
